prov/src/main/java/org/bouncycastle/jce/provider/X509LDAPCertStoreSpi.java

```diff
@@ -408,7 +466,8 @@ public Collection engineGetCRLs(CRLSelector selector)
     private Set search(String attributeName, String attributeValue,
                        String[] attrs) throws CertStoreException
     {
-        String filter = attributeName + "=" + attributeValue;
+        String filter = attributeName + "=" + filterEncode(attributeValue);
+        System.out.println(filter);
```

Comment:
Delete this print.
Check here https://github.com/lineaje-labs-gos/bc-java/commit/3d99e28a9cc2761eecd35460e8ae1b4a93d35d3d where in the main branch, it was commented out

Reply:
removed print